src/assets/styles/styles/custom/_tables.scss

```diff
@@ -83,7 +103,13 @@
     }
 
     .selection {
-      background-color: $selection-color !important;
```

Comment:
pewnie teraz będzie można wywalić selection color ze zmiennych, trzeba to sprawdzić

Reply:
wyrzuciłam, nie było już potrzebne